src/main/java/edu/harvard/iq/dataverse/DatasetField.java

```diff
@@ -420,6 +420,10 @@ public Dataverse getDataverse() {
         } else if (template != null) {
             return template.getDataverse();
         } else {
+
+            System.out.print("getDataverseException: " + this.datasetFieldType.getDisplayName());
```

Comment:
Should we use logger instead?

Reply:
Yes.

Reply:
Really don't need to log it. It's debug code that I should have removed. Throwing the exception is enough, when this state legitimately occurs.